packages/charts/src/charts/metric/metric-chart.component.tsx

```diff
@@ -115,7 +108,7 @@ export const MetricChart: FC<Props> = ({
   const [tooltipPosition, setTooltipPosition] = useState({ x: 0, y: 0 });
 
   return (
-    <Layout>
+    <Layout positionRelativeToDocument={positionRelativeToDocument}>
```

Comment:
Property suggests that it changes only `position` relative or static. However it is used to configure the whole layout.  I do not get it - why it's required at chart plot level ?